views/event-detail.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layout.html" %}
+
+{% block mainContent %}
+  <div>
+    <p>What: {{event.title}}</p>
+    <p>When: {{event.date}}</p>
+    <p>Where: {{event.location}}</p>
+    <p>With whom: {{event.attending}}</p>
+    <p>Why: {{event.image}}</p>
```

Comment:
Pretty sure I got the image issue fixed -- look for it in my next (shortly
upcoming) pull request.

On Mon, Feb 9, 2015 at 10:21 PM, David Hatch <notifications@github.com>
wrote:

> In views/event-detail.html
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mightyLake_mgt656-2Dspring-2D2015-2Dproject_pull_25-23discussion-5Fr24386633&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=qcJxOJmIpoQX59UDHnw6vYS8QNZlleLqZuSHaMKiP_4&m=-3lMnZX9dYSoT_1tNYy6xfOPaJZk4ECHnLo5aXl-FLk&s=P4aSoCs7vVUvov6h_I6K6a3TSju6oTmQ5DexXXoROyA&e=>
> :
>
> > @@ -0,0 +1,34 @@
> > +{% extends "layout.html" %}
> > +
> > +{% block mainContent %}
> > +  <div>
> > +    <p>What: {{event.title}}</p>
> > +    <p>When: {{event.date}}</p>
> > +    <p>Where: {{event.location}}</p>
> > +    <p>With whom: {{event.attending}}</p>
> > +    <p>Why: {{event.image}}</p>
>
> This should display using an img tag so the actual image appears.
>
> —
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_mightyLake_mgt656-2Dspring-2D2015-2Dproject_pull_25_files-23r24386633&d=AwMCaQ&c=-dg2m7zWuuDZ0MUcV7Sdqw&r=qcJxOJmIpoQX59UDHnw6vYS8QNZlleLqZuSHaMKiP_4&m=-3lMnZX9dYSoT_1tNYy6xfOPaJZk4ECHnLo5aXl-FLk&s=FW7uU-fVWrbpyUcOPWMBMmrAfcUCrRDR2-cyITi0sP4&e=>
> .
>